### PR TITLE
IGAPP-497: Snackbar on Failed Refresh

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Comment by Taggotty** [comment]
Not sure if that is what this issue wants, but it shows the snackbar if refresh should not be possible.

File: native/src/routes/Settings.tsx [hunk 1]
```diff
@@ -100,14 +98,6 @@ const Settings = ({ navigation, t, languageCode, cityCode, theme }: PropsType):
 
   const ThemedItemSeparator = () => <ItemSeparator theme={theme} />
 
-  const showSnackbar = (key: string) =>
```
Comment (Taggotty) [inline native/src/routes/Settings.tsx:103]: was never used

**Review by klinzo — CHANGES_REQUESTED** [review]
Sadly this is not what the goal of the issue was. If the refresh fails because of no internet connection or for other reasons, the snackbar should be displayed and the old content is still displayed. Not sure how much effort to implement this is though.

**Comment by Taggotty** [comment]
@klinzo thx for clarifying. Then i'll check out this case. The Payload provider really is confusing in this regard.

**Comment by klinzo** [comment]
> @klinzo thx for clarifying. Then i'll check out this case. The Payload provider really is confusing in this regard.

Definitely! That's why there is also one issue to refactor it. Ugly af :vomiting_face: .

**Review by klinzo — APPROVED** [review]
Only approved to not block the merge later on once it is finished.

**Comment by Taggotty** [comment]
Didn't get any further than that. Probably needs quite a bit of refactoring.

**Pull request closed without merging**